src/documentation/ToolkitDocumentation/ToolkitTopicEntry.tsx

```diff
@@ -46,59 +56,114 @@ const ToolkitTopicEntry = ({
     [setAlternativeIndex]
   );
   const disclosure = useDisclosure();
+  const ref = useRef<HTMLDivElement>(null);
+  const previousAnchor = usePrevious(anchor);
+  const scrollable = useScrollablePanelAncestor();
+  const prefersReducedMotion = usePrefersReducedMotion();
+  const logging = useLogging();
+  const [highlighting, setHighlighting] = useState(false);
+  useEffect(() => {
+    if (previousAnchor !== anchor && active) {
+      logging.log("Activating " + topic.name);
+      disclosure.onOpen();
+      // Delay until after the opening animation so the full container height is known for the scroll.
+      window.setTimeout(() => {
+        if (ref.current && scrollable.current) {
+          scrollable.current.scrollTo({
+            // Fudge to account for the fixed header and to leave a small gap.
+            top: ref.current.offsetTop - 112 - 25,
+            behavior: prefersReducedMotion ? "auto" : "smooth",
+          });
+        }
+        setTimeout(() => {
+          setHighlighting(true);
+          setTimeout(() => {
+            setHighlighting(false);
+          }, 3000);
+        }, 300);
+      }, 150);
+    } else if (previousAnchor !== anchor) {
+      if (ref.current && scrollable.current) {
+        scrollable.current.scrollTo({
+          top: 0,
+          behavior: prefersReducedMotion ? "auto" : "smooth",
+        });
+      }
```

Comment:
It feels like this shouldn't be necessary, but it scrolls to the top of the toolkit area where a topic is linked, but not a specific entry.

Reply:
resolved with commit 10d7bb5